sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_base_client.py

```diff
@@ -13,9 +13,10 @@
 class TextAnalyticsApiVersion(str, Enum):
     """Text Analytics API versions supported by this package"""
 
-    #V3_2_PREVIEW = "v3.2-preview.1"
+    V3_1_PREVIEW_3 = "v3.1-preview.3"
```

Comment:
nit: do we want to mention either in the name, or in a comment that it is gated? one option is `V3_1_PREVIEW_GATED`

Reply:
Since only Healthcare is gated, would that mean we need to create two different variables (one for healthcare and the other for analyze)?